src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,243 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+Dogfood: PRPL is a pattern we believe makes sense for applications that can be
```

Comment:
@petele Had a discussion with Addy about this. The style guide says this: "Remove all Dogfood callouts before making a document publicly visible."  Changing it to a caution and highlighting 'experimental'.